src/app/components/LiveMediaStream/index.tsx

```diff
@@ -80,38 +81,53 @@ const LiveMediaStream = ({ mediaCollection }: Props) => {
 
   return (
     <div css={styles.componentContainer}>
-      <p
-        css={[
-          styles.mediaDescription,
-          warnings && styles.mediaDescriptionGuidance,
-        ]}
-      >
-        <Text size="pica" fontVariant="sansBold" as="span">
-          {short}
-        </Text>{' '}
-        <Text size="pica" fontVariant="sansRegular" as="span">
-          {networkName}
-        </Text>
-      </p>
-      {warnings && (
-        <Text as="p" css={styles.guidanceMessage}>
-          {warnings.warning_text}
-        </Text>
-      )}
       <button
         type="button"
         onClick={() => handleClick()}
         data-testid="watch-now-button"
-        css={[showMedia ? styles.hideComponent : styles.playButton]}
+        css={styles.mediaButton}
       >
         <Text
-          css={styles.playButtonText}
-          size="greatPrimer"
+          size="pica"
           fontVariant="sansBold"
+          as="span"
+          css={[
+            styles.mediaDescription,
+            warnings && styles.mediaDescriptionGuidance,
+          ]}
+          className="hoverStylesText"
         >
-          <PlayIcon />
-          {watchNow}
+          <Text size="pica" fontVariant="sansBold" as="span">
+            {short}
+          </Text>{' '}
+          <VisuallyHiddenText>, </VisuallyHiddenText>
```

Comment:
If this is the comma after the media title, it should be visually displayed e.g. we don't need the vh class